src/main/java/org/sonarlint/intellij/actions/SonarLintToolWindow.java

```diff
@@ -217,22 +216,19 @@ public void showIssueLocations(LiveIssue liveIssue) {
   }
 
 
-  public void show(LocalHotspot remoteHotspot) {
-    //TODO plug in when OPEN in IDE feature modified
-//    activeHotspot = liveHotspot;
-    if (getToolWindow() == null) {
-      // can happen if we try to show while the project is opening
-      // we will show the hotspot when the tool window is built
-      return;
+  public boolean trySelectSecurityHotspot(String securityHotspotKey) {
+    var foundHotspot = liveSecurityHotspots.stream().filter(sh -> securityHotspotKey.equals(sh.getServerFindingKey())).findFirst();
```

Comment:
I noticed this from Mithil's branch, I'm not sure we should keep a list of live findings in the tool window (there is a logic to merge etc, I think it's too much for a UI component). In your case you could rely on the `OnTheFlyFindingsHolder` from `ShowSecurityHotspotCallable`.

Reply:
This is something I removed on my refactoring, maybe I can take care of that on my rebase later?

Reply:
OK, so I let you handle the change @nquinquenel